app/bundles/ReportBundle/Views/FormTheme/Report/filter_selector_row.html.twig

```diff
@@ -0,0 +1,33 @@
+{% block filter_selector_row %}
```

Comment:
Custom form themes should have `_` in front of the block name. ie `_filter_selector_row` or it will never be used.